distribution/docker/build.gradle

```diff
@@ -324,6 +362,8 @@ subprojects { Project subProject ->
 
     final Architecture architecture = subProject.name.contains('aarch64-') ? Architecture.AARCH64 : Architecture.X64
     final boolean oss = subProject.name.contains('oss-')
+    // We can ignore Iron Bank at the moment as we don't
```

Comment:
Just so I understand, we are effectively not building the iron bank image here becuase we have not added a corresponding "export' project for it, yes?

Reply:
Yes, there's no export task, and there won't be without a way to automatically build the image. That would require simulating the Iron Bank image build process.